### PR TITLE
Configure TLS with letsencrypt

### PR DESCRIPTION
**Pull request by yatzek**
Fixes: #2 
Now available at: https://ci.cloudfoundry.org

### CODE REVIEW

File: config/concourse/_ytt_lib/concourse/rendered.yml [hunk 1]
```diff
@@ -43,6 +43,9 @@ spec:
   - name: atc
     port: 80
     targetPort: atc
+  - name: atc-tls
```
Comment (StefanWutz) [inline config/concourse/_ytt_lib/concourse/rendered.yml:46]: @yatzek Is this the correct file? Shouldn't it be added in an overlay file?

**Comment by rkoster** [comment]
Wouldn't https://bosh.ci.cloudfoundry.org/ be more future proof? cc @ramonskie @christopherclark

File: config/concourse/_ytt_lib/concourse/rendered.yml (hunk 1 above)
Comment (yatzek) [reply config/concourse/_ytt_lib/concourse/rendered.yml:46]: This should be fine, the actual change is in the "build/concourse/values.yml" file and the build step produces "config/concourse/_ytt_lib/concourse/rendered.yml" file, there is no need to do any ytt mods here.

**Comment by yatzek** [comment]
Url changed, now available at: https://bosh.ci.cloudfoundry.org

**Pull request merged**